### PR TITLE
CMP-2972: added SPO 0.8.6 release notes

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/CMP-2972

Link to docs preview:
In progress

QE review:
- [ ] QE has approved this change.

Additional information:
This update does not change the meaning of the docs; no QE review required.

### CODE REVIEW

File: security/security_profiles_operator/spo-release-notes.adoc [hunk 1]
```diff
@@ -12,6 +12,15 @@ These release notes track the development of the Security Profiles Operator in {
 
 For an overview of the Security Profiles Operator, see xref:../../security/security_profiles_operator/spo-overview.adoc#[Security Profiles Operator Overview].
 
+[id="spo-release-notes-0-8-6"]
+== Security Profiles Operator 0.8.4
```
Comment (mburke5678) [inline security/security_profiles_operator/spo-release-notes.adoc:16]:
```suggestion
== Security Profiles Operator 0.8.6
```

**Comment by mburke5678** [comment]
@sheriff-rh One comment. Otherwise LGTM

File: security/security_profiles_operator/spo-release-notes.adoc (hunk 1 above)
Comment (sheriff-rh) [reply security/security_profiles_operator/spo-release-notes.adoc:16]: Thank you!

**Pull request merged**